Doc/whatsnew/3.8.rst

```diff
@@ -394,6 +394,9 @@ Deprecated
 
   (Contributed by Serhiy Storchaka in :issue:`33710`.)
 
+* The :meth:`~isAlive()` method of :class:`threading.Thread` has been deprecated.
```

Comment:
If you add "threading.Thread" to isAlive(), it's still displayed as "isAlive()" but the link can be clicked :-)

```suggestion
* The :meth:`~threading.Thread.isAlive()` method of :class:`threading.Thread` has been deprecated.
```